local_settings.sh.in

```diff
@@ -88,6 +88,11 @@ relabel_files() {
     if [ $QUIET -ne 0 ]; then
           opts="-v"
     fi
+    # Starting SELinux 3.4, we can multithread
+    if [ "$(setfiles -h 2>&1 | grep '\[\-[a-zA-Z]*T')" ]; then
+          # "-T 0" will use all available cores.
+          opts="${OPTS} -T 0"
```

Comment:
Shouldn't {OPTS} be in lowercase, here? It doesn't seem to preserve previous additions otherwise, in a quick local test.

Reply:
Woops, good catch! Corrected :)